packages/docs/stories/components/select.stories.ts

```diff
@@ -0,0 +1,404 @@
+  name: 'Multiple with Grouping Options',
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('select', 'multiple'),
+      },
+    },
+  },
+  render: () => html`
+    <syn-select value="option-1 option-2 option-3" multiple clearable>
+      <syn-optgroup label="Option">
+        <syn-option value="option-1">Option</syn-option>
+        <syn-option value="option-2">Option</syn-option>
+      </syn-optgroup>
+      <syn-optgroup label="Option">
+        <syn-option value="option-3">Option</syn-option>
+      </syn-optgroup>
+    </syn-select>
+  `,
+};
+
+export const Sizes: Story = {
+  name: 'Sizes',
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('select', 'size'),
+      },
+    },
+  },
+  render: () => html`
+    <syn-select placeholder="Small" size="small">
+      <syn-option value="option-1">Option 1</syn-option>
+      <syn-option value="option-2">Option 2</syn-option>
+      <syn-option value="option-3">Option 3</syn-option>
+    </syn-select>
+
+    <br />
+
+    <syn-select placeholder="Medium" size="medium">
+      <syn-option value="option-1">Option 1</syn-option>
+      <syn-option value="option-2">Option 2</syn-option>
+      <syn-option value="option-3">Option 3</syn-option>
+    </syn-select>
+
+    <br />
+
+    <syn-select placeholder="Large" size="large">
+      <syn-option value="option-1">Option 1</syn-option>
+      <syn-option value="option-2">Option 2</syn-option>
+      <syn-option value="option-3">Option 3</syn-option>
+    </syn-select>
+  `,
+};
+
+export const PrefixSuffixTextAndIcons: Story = {
+  name: 'Prefix Suffix Text and Icons',
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('select', 'prefix'),
+      },
+    },
+  },
+  render: () => html`
+    <syn-select placeholder="Small" size="small" clearable>
+      <syn-icon name="wallpaper" slot="prefix"></syn-icon>
+      <syn-option value="option-1">Option 1</syn-option>
+      <syn-option value="option-2">Option 2</syn-option>
+      <syn-option value="option-3">Option 3</syn-option>
+    </syn-select>
+    <br />
+    <syn-select placeholder="Medium" size="medium" clearable>
+      <syn-icon name="wallpaper" slot="prefix"></syn-icon>
+      <syn-option value="option-1">Option 1</syn-option>
+      <syn-option value="option-2">Option 2</syn-option>
+      <syn-option value="option-3">Option 3</syn-option>
+    </syn-select>
+    <br />
+    <syn-select placeholder="Large" size="large" clearable>
+      <syn-icon name="wallpaper" slot="prefix"></syn-icon>
+      <syn-option value="option-1">Option 1</syn-option>
+      <syn-option value="option-2">Option 2</syn-option>
+      <syn-option value="option-3">Option 3</syn-option>
+    </syn-select>
+  `,
+};
+
+export const CustomTags: Story = {
+  name: 'Custom Tags',
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('select', 'gettag'),
```

Comment:
There seems to be a typo in our doc token:. The s is missing:

![image](https://github.com/synergy-design-system/synergy-design-system/assets/115139104/1a1edfd3-1f04-498a-b59d-3c8ee3f58b7c)